templates/clock-deployment.yaml

```diff
@@ -46,8 +46,7 @@ spec:
             - --timeout=180s
         {{ end }}
       containers:
-        - args:
-            - ./scripts/start.clock.sh
+        - args: ["./scripts/start.clock.sh"]
           env:
             - name: RAILS_ENV
```

Comment:
don't we need to pass the SIDEKIQ_BILLING env var to this deployment as well? 🤔

Reply:
yes because clock is enqueuing billing job, its better to have it "everywhere"